ballerina/Ballerina.toml

```
@@ -9,23 +9,23 @@ icon = "icon.png"
 license = ["Apache-2.0"]
 distribution = "2201.10.0"
 
-[platform.java17]
+[platform.java21]
 graalvmCompatible = true
 
-[[platform.java17.dependency]]
+[[platform.java21.dependency]]
 groupId = "io.ballerina.stdlib"
 artifactId = "xslt-native"
 version = "2.7.0"
```

Should we bump the version?
```suggestion
version = "2.7.1"
```

---

We can do once we merge to master. We need to do for all other repos as well.